python/setup.py

```diff
@@ -0,0 +1,60 @@
+"""
+Setup script to install the FastFEC python package
+"""
+
+import os
+from distutils.core import Extension
+
+from setuptools import setup
+
+# get current directory
+CURRENT_DIR = os.getcwd()
+PARENT_DIR = os.path.dirname(CURRENT_DIR)
+
+fastfec_library = Extension(
+    "fastfec_lib",
+    [
```

Comment:
This is so ugly I wonder if we should just map the array with the os join

Reply:
yeah that's better